z-schema/index.d.ts

```diff
@@ -90,7 +132,19 @@ declare namespace ZSchema {
          */
         validate(json: any, schema: any, callback: (err: any, valid: boolean) => void): void;
 
+        /**
+         * Returns an Error object for the most recent failed validation, or null if the validation was successful.
+         */
         getLastError(): SchemaError;
-        getLastErrors(): SchemaError[];
+
+        /**
+         * Returns the error details for the most recent validation, or undefined if the validation was successful.
+         * This is the same list as the SchemaError.details property.
+         */
+        getLastErrors(): SchemaErrorDetail[];
     }
-}
\ No newline at end of file
+}
+
+declare module "z-schema" {
```

Comment:
If you're writing in a module declaration, namespaces aren't global. But you could declare a namespace to be merged with the class:
```ts
declare namespace Validator {
    interface SchemaError extends Error { ... }
}
declare class Validator { ... }
export = Validator;
```
then:
```ts
import Validator = require("z-schema");
var error: Validator.SchemaError = new Validator(options).getLastError();
```